e2e/tests/functional/recentObjects.e2e.spec.js

```diff
@@ -252,6 +252,30 @@ test.describe('Recent Objects', () => {
         // Assert that the list is empty
         expect(await recentObjectsList.locator('.c-recentobjects-listitem').count()).toBe(0);
     });
+    test("Clears the recent objects and disables the button", async ({ page }) => {
+        test.slow();
```

Comment:
We can remove this `test.slow()` here. This line will double the default timeout time (in the case of a long-running test).